met-web/src/components/Login/index.tsx

```diff
@@ -1,9 +1,16 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import UserService from "../../services/UserServices";
 import { Grid } from "@mui/material";
 import Button from "@mui/material/Button";
+import { useAppSelector, useAppDispatch } from "../../hooks";
 
 const Login = () => {
+  const [isLoggedIn, setIsLoggedIn] = useState<boolean>(false);
+
+  useEffect(() => {
+    setIsLoggedIn(UserService.isLoggedIn);
```

Comment:
useEffect always needs seconds parameter. Also user shouldn't be on Login page if user is not logged in. It seems out of place to check (that user is logged in) in this component.